maqs-utilities/src/main/java/com/cognizantsoftvision/maqs/utilities/logging/IFileLogger.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 (C) Cognizant SoftVision, All rights Reserved
+ */
+
+package com.cognizantsoftvision.maqs.utilities.logging;
+
+/**
+ * The File Logger interface class.
+ */
+public interface IFileLogger extends ILogger {
+
+  /**
+   * Gets the FilePath value.
+   *
+   * @return returns the file path
+   */
+  String getFilePath();
+
+  /**
+   * Gets the Message Type value.
+   *
+   * @return The Message Type.
+   */
+  MessageType getMessageType();
+
+  /**
+   * Gets the Directory Path.
+   *
+   * @return Returns the Directory
+   */
+  String getDirectory();
+
+  /**
+   * Sets the FilePath value.
+   *
+   * @param path sets the file path
+   */
+  void setFilePath(String path);
+
+  /**
+   * Gets the File Name value.
+   *
+   * @return Returns the File Name.
+   */
+  String getFileName();
+
+  /**
+   * Logs the message.
+   * @param message The message text.
+   * @param args String format arguments
+   */
+  void logMessage(String message, Object... args);
```

Comment:
Maybe my Java is just rusty, but I don't think we want the logMessage functions in this interface as they are already part of the parent interface.